src/App.css

```diff
@@ -1,4 +1,4 @@
-#root {
```

Comment:
This files doesn't seem to be used. Let's remove.

Reply:
Ok.